.github/workflows/docs-build.yml

```diff
@@ -14,7 +14,9 @@ jobs:
   docs-preview:
     uses: elastic/docs-builder/.github/workflows/preview-build.yml@main
     with:
-      path-pattern: docs/**
+      path-pattern: |
+        docs/**
+        dev_docs/**
```

Comment:
The dev_docs are built with a separate system, and won't be able to work these files.

We have a validation step for dev_docs at https://github.com/elastic/kibana/blob/main/.buildkite/pipelines/pull_request/check_next_docs.yml, but they don't currently support previewing changes.